CLAUDE.md

```
@@ -385,6 +386,15 @@ const { t } = useTranslation(['navigation', 'common']);
 <span>GitHub PRs</span>                          // ❌ WRONG
 ```
 
+**Error messages with substitution:**
+```tsx
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)